src/exec.ts

```diff
@@ -18,7 +19,8 @@ export class Exec {
 
     // TODO: make command an array and support multiple args
     public async exec(namespace: string, podName: string, containerName: string, command: string,
-                      stdout: stream.Writable | any, stderr: stream.Writable | any, stdin: stream.Readable | any,
+                      stdout: stream.Writable | null, stderr: stream.Writable | null, stdin: stream.Readable | null,
+                      statusCallback: (status: V1Status) => void,
```

Comment:
Usage of `statusCallback` is non-obvious, even with the exec sample.  Can we provide any documentation on this?

Reply:
Added JSDoc.